tests/unit/state-getters.spec.ts

```diff
@@ -357,6 +363,10 @@ describe('SP/GP correction getters', () => {
       },
       shareStructure: {
         shareClasses: []
+      },
+      authInfo: {
+        contact: {},
+        folioNumber: ''
```

Comment:
Do we still need these changes?

Reply:
It was used earlier, but not anymore. They are now reverted.